README.md

```diff
@@ -8,6 +8,7 @@
 [![action-bumpr supported](https://img.shields.io/badge/bumpr-supported-ff69b4?logo=github&link=https://github.com/haya14busa/action-bumpr)](https://github.com/haya14busa/action-bumpr)
 
 This GitHub Action runs [protolint](https://github.com/yoheimuta/protolint) with [reviewdog](https://github.com/reviewdog/reviewdog).
+By default, action-protolint uses the `.protolint.yaml` configuration file if it exists.
```

Comment:
I assume that protolint would work in that manner, so action-protolint shouldn't need to specify the `.protolint.yaml` file located in the current working directory. Have you confirmed whether protolint failed to locate that file?

> protolint will automatically search a current working directory for the config file by default and successive parent directories all the way up to the root directory of the filesystem. And it can search the specified directory with -config_dir_path flag. It can also search the specified file with --config_path flag.
> https://github.com/yoheimuta/protolint#configuring

Reply:
It looks like the problem might be related to the quotes around ${INPUT_PROTOLINT_FLAGS}. I removed the code check for the .protolint.yaml config, protolint can still find the configuration file.